chef/cookbooks/ceph/recipes/mon.rb

```diff
@@ -19,8 +19,9 @@
 include_recipe "ceph::conf"
 
 service_type = node["ceph"]["mon"]["init_style"]
+address = node["crowbar"]["network"]["public"] || node["hostname"]
```

Comment:
This is what happens when you eyeball it while waiting for your cloud to be setup :D

Thanks for the tips @vuntz will redo once I can test the whole thing!